ext/dapr-ext-workflow/setup.cfg

```diff
@@ -25,6 +25,7 @@ packages = find_namespace:
 include_package_data = True
 install_requires =
     dapr-dev >= 1.9.0rc1.dev
+    durabletask >= 0.1.0a2
```

Comment:
Do we need to update this again after your latest fix to make `durabletask-python` compatible with 3.7 or does this version already have that support?

Reply:
Yes it seems you need to release 0.1.0a3 and then update this here. The version with 3.7 compatibility has not been released yet.

Reply:
0.1.0a2 durabletask-python also has 3.7 support. But, had marked 0.1.0a3 in durabletask-python, as wanted to keep Pypi and source code on github in sync. As, 0.1.0a2 had Pypi support even when 3.7 compliant PR was not merged.
Need to automate the release process in durabletask-python, I think then this mis-match issue will not be there.